superdesk/publish/publish_queue.py

```diff
@@ -101,7 +101,7 @@ class PublishQueueResource(Resource):
     }
 
     etag_ignore_fields = ['moved_to_legal']
-    datasource = {'default_sort': [('_created', -1)]}
+    datasource = {'default_sort': [('_created', -1), ('subscriber_id', 1), ('published_seq_num', -1)]}
```

Comment:
I'd like to get this change merged and onto UAT then create the index manually, Once we're happy with it add the index creation to app_initialize.py